lib/Onyx.js

```diff
@@ -909,6 +909,21 @@ function hasPendingMergeForKey(key) {
     return Boolean(mergeQueue[key]);
 }
 
+/**
+ * For objects, the key for null values needs to be removed from the object to ensure the value will get removed from storage completely.
+ * On native, SQLite will remove top-level keys that are null. To be consistent, we remove them on web too.
```

Comment:
It might be better to expand this comment a little more to explain:

1. What the default behavior is on Web since it's not using SQLite
2. Why that default behavior isn't consistent with what SQLite does
3. Why the behavior needs to be consistent for all platforms

It could be something like this (though I don't know that these answers are 100% correct).

>  On native, SQLite removes top-level keys that have null values. This is desired because it decreases the amount of data stored in memory and on disk. On web, IndexedDB will keep the top-level keys along with a null value and this uses up storage and memory. This method will ensure that keys for null values are removed before an object is persisted to disk so that all platforms are storing the data in the same efficient way.